src/main/java/webApp/webController.java

```diff
@@ -0,0 +1,47 @@
+package webApp;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.*;
+import survey.Survey;
+
+import java.util.ArrayList;
+import java.util.stream.Collectors;
+
+@Controller
+public class webController {
+    @Autowired
```

Comment:
You are missing the repository for the questions.

Reply:
Yes, checked, realized it's not required. Thanks